docs/meshstack.billing-configuration.md

```diff
@@ -511,40 +511,28 @@ let example
 
-### Multi-Currency
+If your meshStack is hosted in a private environment and has permission to make external requests, you can activate the default converter using the toggle button on the Settings page, as mentioned earlier. If external requests are restricted, you have the option to enable meshStack to communicate with the Frankfurter API through Firewall rules. Alternatively, if no conversion is desired for chargebacks, you can choose not to apply any conversion by simply turning off togle button in the Financial Tab of the Settings page.
 
-Sometimes Cloud Foundation teams encounter situations in which they need to process billing or chargeback in multiple currencies. For example GCP might charge your organization for cloud consumption in EUR while Azure charges consumption in USD.
 
-In case standardisation on a single currency is not possible, meshStack supports chargeback in multiple currencies. The
-following product funcitonalities fully support multi-currency scenarios:
+### Multi-Currency
 
-- **Private Cloud billing**: Partners can define products and usage rates in any currency.
-- **Public Cloud billing**: meshStack preserves the original currency when importing cost and consumption data from the cloud provider.
-- *OSB Services** (old meshMarketplace): Service owners can define prices for their services using the OSB Catalog in any currency.
-- **Tenant Usage Reports**: Tenant usage reports support line items in multiple currencies and aggregates net amounts in each currency separately.
-- **Chargeback Statements**: Chargeback statements support multiple currencies and aggregate net amounts in each currency separately.
-- **Tenant Fees**: Partners can define tenant fees in any currency.
-- **Discounts**: Discounts support any currency. Discounts calculated from a source amount (e.g. percentage discounts) use the same currency as the source amount.
+If you decide not to use currency conversion and prefer to have multi-currency in chargebacks, simply disable the toggle on the Financial Tab in the Settings page.
 
 The following product functionalitities do currently not support multi-currency scenarios:
 
-- **Detailed Tenant Usage Report**: Detailed tenant usage reports (available for OpenShift, OpenStack, Cloud Foundry and
-OSB Services) offer additional detail on a Tenant Usage Report. However, the net amount aggregation assumes all line items
-  are in EUR. This can create confusion if the underlying Tenant Usage Report uses a different or multiple currencies.
-- **Payment Methods**: Amounts specified on payment methods are currently assumed to be in EUR.
+- **Detailed Tenant Usage Report**: Detailed tenant usage reports (available for OpenShift, OpenStack, Cloud Foundry and OSB Services) offer additional detail on a Tenant Usage Report. However, the net amount aggregation assumes all line items are in EUR. This can create confusion if the underlying Tenant Usage Report uses a different or multiple currencies.
```

Comment:
```suggestion
- **Detailed Tenant Usage Report**: Detailed tenant usage reports (available for OpenShift, OpenStack, Cloud Foundry and OSB Services) offer additional detail on a Tenant Usage Report. However, the net amount aggregation assumes all line items are in EUR. This can create confusion if the underlying Tenant Usage Report uses different or multiple currencies.
```